README.md

```diff
@@ -1,17 +1,117 @@
 
-## Workflow for new channels
+## Contents
 
-1) Head over to the [skeleton repo](https://github.com/laravel-notification-channels/skeleton) & press "Use this template" to create a repo from the skeleton.
-2) Use find/replace to replace all of the placeholders with the correct values (package name, author name, email, etc).
-3) Implement to logic for the channel & add tests.
-4) Fork this repo, add it as a remote and push your new channel to a branch.
-5) Submit a new PR against this repo for review.
+- [Installation](#installation)
+	- [Setting up the Google Hangouts Chat service](#setting-up-the-google-hangouts-chat-service)
+- [Usage](#usage)
+	- [Available Message methods](#available-message-methods)
+- [Changelog](#changelog)
+- [Testing](#testing)
+- [Security](#security)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
 
-Take a look at our [FAQ](http://laravel-notification-channels.com/) to see our small list of rules, to provide top-notch notification channels.
+
+## Installation
+
+Install this package with Composer:
+
+```bash
+composer require laravel-notification-channels/hangouts-chat
+```
+
+
+### Setting up the Google Hangouts Chat service
+
+In order to send messages using bots, you need to [setup a webhook for a room](https://developers.google.com/hangouts/chat/how-tos/webhooks#send_messages_to_the_chat_room).
+
+## Usage
+Now you can use the channel in your `via()` method inside the notification:
+
+``` php
+use NotificationChannels\HangoutsChatChannel;
+use NotificationChannels\HangoutsChatMessage;
+use Illuminate\Notifications\Notification;
+
+class ProjectCreated extends Notification
+{
+    public function via($notifiable)
+    {
+        return ['hangoutsChat'];
+    }
+
+    public function toHangoutsChat$notifiable)
+    {
+        return HangoutsChatMessage::create()
+            ->data([
+               'text' => 'It is a test message!'
+            ]);
+    }
+}
+```
+Please look at [Hangouts Chat Message Formats](https://developers.google.com/hangouts/chat/reference/message-formats) to understand how send the messages.
+
+In order to let your Notification know which URL should receive the Webhook data, add the `routeNotificationForHangoutsChat` method to your Notifiable model.
+This method needs to return the URL where the notification Webhook will receive a POST request.
+
+
+```php
+public function routeNotificationForHangoutsChat()
+{
+    return 'https://chat.googleapis.com/v1/spaces/XXXXX-XXXXX/messages?key={key}&token={token}';
+}
+```
+
+You can send to a dynamic Webhook URL on-fly using this syntax:
+```php
+Notification::route('hangoutsChat', 'https://chat.googleapis.com/v1/spaces/XXXXX-XXXXX/messages?key={key}&token={token}')
+                    ->notify(new NotificationClass($params));
+```
+
+### Available Message methods
+
+- `data('')`: Accepts a JSON-encodable value for the Webhook body.
```

Comment:
I feel like it would be a bit easier to use if we provided some simple ways to set the content, with a back of custom json only if you're doing something complex
What do you think?